tools/query-account/src/main.rs

```diff
@@ -0,0 +1,134 @@
+use std::thread::sleep;
+use std::time::Duration;
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use reqwest::{blocking::*, Url};
+use rosetta::models::*;
+
+#[derive(Parser, Debug)]
+#[clap(
+    author = "Concordium Foundation",
+    about = "Client for querying the chain and filter out operations related to a given account.",
+    version
+)]
+struct Args {
+    #[clap(long = "url", help = "URL of Rosetta server.", default_value = "http://localhost:8080")]
+    url:     String,
+    #[clap(
+        long = "network",
+        help = "Network name. Used in network identifier.",
+        default_value = "testnet"
+    )]
+    network: String,
+    #[clap(long = "address", help = "Address of the account.")]
+    address: String,
```

Comment:
Use `AccountAddress` type instead. It implements FromStr so can be used
http://developer.concordium.software/concordium-rust-sdk/id/types/struct.AccountAddress.html#impl-FromStr

Reply:
The project doesn't depend on the SDK so doesn't have that type available.